source/geometries/Next100EnergyPlane.cc

```diff
@@ -207,11 +207,10 @@ namespace nexus {
 			 0, G4ThreeVector(0., 0., transl_z));
 
     G4Tubs* hole_hut_solid =
-      new G4Tubs("HOLE_HUT", 0., hut_int_diam_/2., (hut_hole_length_ + offset)/2.,
```

Comment:
I don't understand why you removed the offset here. We can discuss it online, if it's easier.

Reply:
Using an offset as in this class is usefull to make an G4Union between a a smaller volume into a larger one, since the `offset` part of the smaller volume is included into the larger. But this is not true here because the hole in the hut is larger (higher radious) than the hole in the copper plate. Such union makes the hole inside the copper to be biased, as you can see in the figures
Expected holes:
<img width="460" alt="expected-holes" src="https://user-images.githubusercontent.com/35993868/126967479-25d211fe-bbc9-405e-b873-1aa75659b99a.png">

Holes with the offset between hut and copper:
<img width="460" alt="holes with offset" src="https://user-images.githubusercontent.com/35993868/126967798-cf1a28e9-98fc-48f2-bb26-f1967ebeba9e.png">

As you see in the second figure, part of the hut hole is embedded into the copper plate, which is not correct.